.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required
+dist: trusty
+
+language: python
+python:
+  - 3.5
+
+
+install:
+  # Install code sniffer.
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
```

Comment:
This is a script, and you don't need the `if` so you actually want something like this:
```yml
install:
  # Install code sniffer.
  - pip install pycodestyle
  - pip install --upgrade pycodestyle
  - source ~/.bash_profile
  - source ~/.bashrc
```